www/app.py

```diff
@@ -74,6 +88,20 @@ def temptargets():
     data = json.load(open(json_url))
     return jsonify(data)
 
+@app.route("/cgm")
+def cgm():
+    json_url = os.path.join("/root/myopenaps/monitor/xdripjs/cgm-pill.json")
```

Comment:
This should use myopenaps_dir, not be hardcoded to the default path.

Reply:
Looks like this is incorrect throughout this file.  It appears that @zjedr removed the only uses of myopenaps_dir in 648369222d8939c9a3183045fd5439ff17075869 and replaced them with hard-coded paths.

So if you'd like to defer fixing that to a later PR, I'd also be OK with that.

Reply:
Good suggestions. I updated all hard-coded paths to use myopenaps_dir. Please take another look while I'm testing tonight.